### PR TITLE
old code for review

### PR DESCRIPTION
**Pull request by joyapisi**
add old code from  Microverse Admissions Trial for review

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+      integrity="sha512-rqQltXRuHxtPWhktpAZxLHUVJ3Eombn3hvk9PHjV/N5DMUYnzKPC1i3ub0mEXgFzsaZNeJcoE0YHq0j/GFsdGg=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <title>Personal Portfolio</title>
+  </head>
+  <body>
+    <!-- START NAV -->
+
+    <nav id="navbar" class="nav">
```
Comment (joyapisi) [inline index.html:29]: Please always use the class attribute for multiple elements and avoid using the ID selector for multiple elements. Apply this in other parts of your HTML document. This helps you to have clean code.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.8.2/css/all.min.css"
+      integrity="sha512-rqQltXRuHxtPWhktpAZxLHUVJ3Eombn3hvk9PHjV/N5DMUYnzKPC1i3ub0mEXgFzsaZNeJcoE0YHq0j/GFsdGg=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <title>Personal Portfolio</title>
+  </head>
+  <body>
+    <!-- START NAV -->
```
Comment (joyapisi) [inline index.html:27]: Please leave a space before the START NAV comment to help ensure readability and make it look separate from the previous block of code that it is unrelated to.

**Review by joyapisi — COMMENTED** [review]
Highlights 💯
Great design and UI ✔️
Descriptive Pull request summary✔️
Good commit messages ✔️
Professional Readme file ✔️
Good use of semantic tags ✔️

Required Changes ♻️
1. Please set up HTML-CSS linters to be able to check if you have linter errors in your code.
2. Check the rest of the comments under the review.

**Review by Olamarx — APPROVED** [review]
Hi @joyapisi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good use of GitHub comments 👍🏼
- Good use of inline comments ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
